Packs/Resecurity/Integrations/ResecurityMonitoring/ResecurityMonitoring.yml

```diff
@@ -0,0 +1,104 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: ResecurityMonitoring
+  version: -1
+configuration:
+- defaultvalue: https://app.resecurity.com/
+  display: Your server URL
+  name: url
+  required: true
+  type: 0
+- displaypassword: API Key
+  additionalinfo: The API Key to use for connection
+  name: credentials
+  required: true
+  hiddenusername: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: 'This package allows retrieving asset monitoring results from monitoring tasks that can be configured in Context and Risk platforms.'
+display: 'Resecurity Monitoring'
+name: ResecurityMonitoring
+script:
+  commands:
+  - arguments:
+    - default: true
+      description: Monitor Task ID.
+      isArray: false
+      name: monitor_task_id
+      required: true
+      secret: false
+    - default: false
+      description: Page number. The results will be returned only for specified page if param value is not empty.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: "Page size. Possible range of values: 1 - 50. Default value is 20."
+      isArray: false
+      name: page_size
+      required: false
+      secret: false
+    - default: false
+      description: Limit of the records in dataset. Default value is 1000.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: "Affects which results will be included in dataset. Possible values: 1 - only new results, 2 - last results (default value), 3 - all results"
+      isArray: false
+      name: mode
+      required: false
+      secret: false
+    deprecated: false
+    description: 'Retrieve monitoring results from a specific monitor task'
+    execution: false
+    name: get-task-monitor-results-data-breaches
```

Comment:
Please add `resecurity-` prefix to the command name.

Reply:
+1 @oleyur please handle it and update code accordingly